src/ResourceManager/Compute/Commands.Compute/AvailabilitySets/NewAzureAvailabilitySetCommand.cs

```diff
@@ -86,27 +86,16 @@ public override void ExecuteCmdlet()
                 {
                     Location = this.Location,
                     PlatformUpdateDomainCount = this.PlatformUpdateDomainCount,
-                    PlatformFaultDomainCount = this.PlatformFaultDomainCount,
-#pragma warning disable CS0618 // Type or member is obsolete
-                    Managed = this.Managed.IsPresent ? (bool?) true : null
-#pragma warning restore CS0618 // Type or member is obsolete
+                    PlatformFaultDomainCount = this.PlatformFaultDomainCount
                 };
 
-#pragma warning disable CS0618 // Type or member is obsolete
-                if (this.Managed.IsPresent || !string.IsNullOrEmpty(this.Sku))
-#pragma warning restore CS0618 // Type or member is obsolete
+                if (!string.IsNullOrEmpty(this.Sku))
```

Comment:
What happens for users who are using the Managed parameter and do not supply the Sku parameter?  Is there a default value for Sku that will work for them.  If so, this should be fine, otherwise, we should set a Sku value that will work based on their Managed setting.